src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -235,13 +231,13 @@ def on_initialize_simulants(self, pop_data):
             condition_column = pd.Series(
                 self.initial_state, index=population.index, name=self.state_column
             )
-        self.population_view.update(condition_column)
+        self.population_view.update(pd.concat([condition_column, initial_propensity], axis=1))
 
     @staticmethod
     def assign_initial_status_to_simulants(
         simulants_df, state_names, weights_bins, propensities
     ):
-        simulants = simulants_df[["age", "sex", "initial_child_wasting_propensity"]].copy()
+        simulants = simulants_df[["age", "sex"]].copy()
```

Comment:
Can you add typing to this method. Also the variable names are a bit confusing to me. This function is actually returning a dataframe, but the method calling it acts like it is only returning a single column. I think only returning the column is more correct, but at least adding typing would make this clear.